src/utils/MetaboliteSpectralMatcher.cpp

```diff
@@ -91,106 +85,104 @@ class TOPPMetaboliteSpectralMatcher :
         public TOPPBase
 {
 public:
-    TOPPMetaboliteSpectralMatcher() :
-        TOPPBase("MetaboliteSpectralMatcher", "Find potential HMDB ids within the given mass error window.", false)
-    {
-    }
+  TOPPMetaboliteSpectralMatcher() :
+      TOPPBase("MetaboliteSpectralMatcher", "Find potential HMDB ids within the given mass error window.", false)
+  {
+  }
 
 protected:
 
-    void registerOptionsAndFlags_()
-    {
-        registerInputFile_("in", "<file>", "", "mzML file");
-        setValidFormats_("in", ListUtils::create<String>("mzML"));
-        registerOutputFile_("out", "<file>", "", "mzTab file");
-        setValidFormats_("out", ListUtils::create<String>("csv"));
-
-        // addEmptyLine_();
-        // addText_("Parameters for the accurate mass search can be given in the 'algorithm' part of INI file.");
-        registerSubsection_("algorithm", "Algorithm parameters section");
-    }
-
-    Param getSubsectionDefaults_(const String& /*section*/) const
+  void registerOptionsAndFlags_()
+  {
+    registerInputFile_("in", "<file>", "", "mzML file");
+    setValidFormats_("in", ListUtils::create<String>("mzML"));
+    registerInputFile_("database", "<file>", "CHEMISTRY/MetaboliteSpectralDB.mzML", "mzML file", false);
```

Comment:
maybe add a comment that this file is generally found in OpenMS/share - otherwise it is confusing where this "magic" file comes from. Maybe even add an URL to the comment section where one can download current and previous versions of the file... ?